flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionModelData.java

```diff
@@ -42,12 +43,9 @@
  * <p>This class also provides methods to convert model data from Table to Datastream, and classes
  * to save/load model data.
  */
-public class LogisticRegressionModelData {
-
-    public DenseVector coefficient;
-
+public class LogisticRegressionModelData extends GeneralLinearAlgoModelData {
```

Comment:
`LogisticRegressionModelData` and `LinearRegressionModelData` seems to be different only in a type cast. If it is true, we can move most logics in this class to `GeneralLinearAlgoModelData`. In that case, `LinearRegressionModelData` only needs to specify a generic `T` or pass a `Class<T>` to the abstract `GeneralLinearAlgoModelData`'s constructor or super method.

Reply:
I also tried to do this but failed because in `ModelDataEncoder` we need to construct an instance of `LinearRegressionModelData` and `LogisticRegressionModelData `. If we pass a class type, we may need to go with reflections, which is usually not encouraged.